analytics/analytics.go

```diff
@@ -89,6 +90,8 @@ func Initialize(n *core.IpfsNode, BTFSVersion string) {
 		return
 	}
 
+	statusServerDomain = configuration.StatusServerDomain
```

Comment:
Looks good, you need to change go-btfs-config to v0.1.6 in go.mod since it's merged and tagged.